clang/examples/DPCT/Runtime/cudaGetChannelDesc.cu

```diff
@@ -1,6 +1,7 @@
 void test(cudaChannelFormatDesc *pc) {
+  // TODO: a's type need to be changed to cudaArray_const_t
```

Comment:
Please fix the todo

Reply:
Now the cudaArray_const_t cannot be migrated correctly.